### PR TITLE
add wish list user products and add spinner

### PR DESCRIPTION
**Pull request by Igor-Klein-1988**

### CODE REVIEW

File: public/locales/en.json [hunk 1]
```diff
@@ -1,8 +1,10 @@
 {
-	"404-page": {
-		"content_title": "Opps! We did not find this page",
-		"content_desc": "But you can return to the main page",
-		"link_back_to_home": "Back To Home"
+	"products": {
+		"add_to_cart": "Add To Cart",
```
Comment (maiorovvv) [inline public/locales/en.json]: I think the translation “add to cart”, like in other translations, should be made with a small letter, or vice versa in German and Russian translations, with a capital letter.

File: src/index.scss [hunk 2]
```diff
@@ -57,7 +58,7 @@
 // @import "elements/product-details";
 // @import "elements/blog-details";
 // @import "elements/contact";
-// @import "elements/cart-page";
+@import "./assets/scss/elements/cart-page";
```
Comment (maiorovvv) [inline src/index.scss:61]: 25/10/2023 @Igor-Klein-1988 wrote:"NotFound.module.scss - вот в такой файл со стилями надо подключить". If these are product card styles, then it seems that we agreed that they should be called “cart-page.module.scss” and stored next to the component.

File: src/features/user/wishProducts/ProductList.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+										<tr className="cart__table--body__items" key={product.id}>
+											<Product product={product} />
+										</tr>
+									))}
+								</tbody>
+							</table>
+							<div className="cart__continue__shopping d-flex justify-content-between">
+								<a className="cart__continue__shopping--link" href="index.html">
+									Continue shopping
+								</a>
```
Comment (maiorovvv) [inline src/features/user/wishProducts/ProductList.tsx:53]: I think this needs translation into other languages

File: src/features/user/wishProducts/ProductList.tsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+								<a className="cart__continue__shopping--clear" href="shop.html">
+									View All Products
```
Comment (maiorovvv) [inline src/features/user/wishProducts/ProductList.tsx]: I think this needs translation into other languages

**Review by maiorovvv — CHANGES_REQUESTED** [review]
A few notes.

File: src/index.scss (hunk 2 above)
Comment (Igor-Klein-1988) [reply src/index.scss:61]: no. A told about it in call today

File: public/locales/ru.json [hunk 5]
```diff
@@ -1,8 +1,10 @@
 {
-	"404-page": {
-		"content_title": "Упс! Мы не нашли эту страницу",
-		"content_desc": "Но вы можете вернуться на главную страницу",
-		"link_back_to_home": "Вернуться на главную"
+	"products": {
+		"add_to_cart": "добавить в корзину",
```
Comment (DenysBabenko29) [inline public/locales/ru.json:3]: need to be in capital letters

File: src/features/user/wishProducts/Products/types/Product.ts [hunk 6]
```diff
@@ -0,0 +1,10 @@
+export default interface Product {
```
Comment (DenysBabenko29) [inline src/features/user/wishProducts/Products/types/Product.ts:1]: name IProduct will be better

**Review by maiorovvv — APPROVED** [review]

File: public/locales/ru.json (hunk 5 above)
Comment (Igor-Klein-1988) [reply public/locales/ru.json:3]: I decided to do everything with a small letter here

**Pull request merged**